portfolio/src/main/java/com/google/sps/servlets/DeleteComments.java

```diff
@@ -0,0 +1,55 @@
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.sps.servlets.DataServlet;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import java.io.IOException;
+import com.google.gson.Gson;
```

Comment:
line 24 should be the first ```import```

Reply:
wait im wrong it should be switched with line 23 to be in alphabetical order, sorry.

Reply:
thanks! updated!

Reply:
Chris wrote a [bash script](https://gist.github.com/ccondit/55f6760a001a9aa4e2167e81ea9bd73b) tool to help with this in the future.

Reply:
Can you try running this file through the bash script? There are now unused imports (which it will hopefully remove) and the formatting looks slightly off...